project/sbt/sbt

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+SBT_OPTS="-Xms512M -Xmx1536M -Xss1M -XX:+CMSClassUnloadingEnabled -XX:MaxPermSize=256M"
```

Comment:
I remember getting OOM when heap size isn't specified.  Actually this file just comes from copy/pasting http://www.scala-sbt.org/0.13/tutorial/Manual-Installation.html.  Didn't put much thought into each setting, but I think we can trust the official site.